tools/didc/src/main.rs

```diff
@@ -232,8 +242,17 @@ fn main() -> Result<()> {
             format,
             annotate,
         } => {
+            let blob = match blob {
+                Some(blob) => blob,
+                None => io::read_to_string(io::stdin())?,
+            };
             let bytes = match format.as_str() {
```

Comment:
Files with hex often contain whitespace.  Some editors like to add a newline at the end.  xxd output is easier to view if wrapped at 60 lines rather than being on one gigantic line.  `echo "abba" | didc decode` may complain about abba not being valid candid but it seems silly for it to complain about the trailing whitespace.